layouts/partials/api/permissions.html

```diff
@@ -0,0 +1,41 @@
+{{ $context := .context }}
+{{ $endpoint := .endpoint }}
+{{ $securitySchemes := .securitySchemes }}
+{{ $tag := .tag }}
+
+{{ $permissionSet := index $endpoint.action "x-permission" }}
+{{ $operator := $permissionSet.operator }}
+{{ $permissions := $permissionSet.permissions }}
+
+{{ $authZ := slice }}
+{{ range $endpoint.action.security }}
+  {{ range $sec_key, $sec_val := . }}
+    {{ if eq $sec_key "AuthZ" }}
+      {{ $authZ = . }}
+    {{ end }}
+  {{ end }}
+{{ end }}
+
+{{ with $permissions }}
+  {{ if eq (len .) 1 }}
+    This endpoint requires the <code>{{ delimit . ", "}}</code> permission.
+  {{ else }}
+    This endpoint requires
+    {{ if eq $operator "OR" }}
+      any
+    {{ else }}
+      all
+    {{ end }}
+    of the following permissions:
+    {{ range $permissions }}
+      <li><code>{{ . }}</code></li>
+    {{ end }}
+    <br/>
+  {{ end }}
+{{ end }}
+
+
+{{ with $authZ }}
+  {{ $slug := (replaceRE " " "-" ($tag.name | humanize | lower)) }}
+  <p>OAuth apps will require the <code>{{ delimit . ", "}}</code> authorization <a href="{{ "api/latest/scopes/" | absLangURL }}#{{ $slug }}">scope</a> to access this endpoint.</p>
```

Comment:
```suggestion
  <p>OAuth apps require the <code>{{ delimit . ", "}}</code> authorization <a href="{{ "api/latest/scopes/" | absLangURL }}#{{ $slug }}">scope</a> to access this endpoint.</p>
```
We use present tense in the public docs.